tests/unit/tuf_repository_service_worker/test_repository.py

```diff
@@ -230,23 +230,17 @@ def test_bump_expiry(self, monkeypatch, test_repo):
             "get_repository_settings",
             lambda *a, **kw: fake_settings,
         )
-        fake_time = datetime.datetime(2019, 6, 16, 9, 5, 1)
-        fake_datetime = pretend.stub(
-            now=pretend.call_recorder(lambda: fake_time)
-        )
-        monkeypatch.setattr(
-            "repository_service_tuf_worker.repository.datetime", fake_datetime
-        )
+        fake_datetime = mocked_datetime
         fake_role = pretend.stub(
-            signed=pretend.stub(expires=fake_time),
+            signed=pretend.stub(expires=fake_datetime.now()),
         )
 
         result = test_repo._bump_expiry(fake_role, "root")
         assert result is None
         assert fake_role.signed.expires == datetime.datetime(
             2023, 6, 15, 9, 5, 1
         )
-        assert fake_datetime.now.calls == [pretend.call()]
+        assert fake_datetime.now.calls == [pretend.call(), pretend.call()]
```

Comment:
I see that by introducing a new `mocked_date` adds a new `pretend.call()` because of line 235.
I don't see how this can be fixed.
Do you @kairoaraujo?

Reply:
I would say that we could remove the `fake_datetime.now.calls` check.
I think it is less confusing for contributors.

Reply:
Thank you for your answer. So I deleted this assert.